APE0.rst

```diff
@@ -0,0 +1,360 @@
+to approve the removal. In the period between removal/resignation and new
+appointment, the Coordination Committee will temporarily take over the
+responsibilities of the Ombudsperson, should the need arise.
+
+The Voting Members
+------------------
+
+Role
+^^^^
+The Voting Members are the group of trusted individuals who operate the Astropy
+Project on behalf of the community. They have authority over the Astropy
+Project’s technical resources, including the Astropy Project website itself, the
+Astropy GitHub organization and repositories, the issue tracker, and all Astropy
+communication channels. In practice, much of this authority is passed to the
+Coordination Committee via the voting processes described in this Charter.
+
+They also assume many roles required to achieve the Project's goals, especially
+those that require a high level of trust. Collectively, they make decisions that
+shape the future of the Project.
+
+Voting Members are expected to act as role models for the community and
+custodians of the Project, on behalf of the community and all those who rely on
+Astropy. They will act as representatives of the Astropy Project, where
+necessary, including in online discussions or at official Astropy events.
+
+Powers
+^^^^^^
+Voting Members may participate in formal votes on:
+* adding new Voting Members,
+* removing Voting Members,
+* electing Coordination Committee members,
+* removing Coordination Committee members,
+* appointment or removal of the Ombudsperson,
+* changes to this Charter,
+* other matters for which the Coordination Committee believes a vote is
+  appropriate.
+
+The mechanism, timeline, and criteria for a decisive vote are specified in the
+respective sections of this document in the first six cases and by the
+Coordination Committee in the last case.
+
+Membership
+^^^^^^^^^^
+Voting Members of the Astropy Project must demonstrate:
+* a good grasp of the philosophy of the Astropy Project,
+* a solid track record of being constructive and helpful,
+* significant contributions to the Project's goals, in any form,
+* a willingness to dedicate time to improving the Project,
+* a willingness to recruit, train, and nominate new team members.
+
+Voting Membership acknowledges sustained and valuable efforts that align well
+with the philosophy and the goals of the Astropy Project.
+
+Initial Membership
+^^^^^^^^^^^^^^^^^^
+Anyone who satisfies any of the following criteria at the date of acceptance of
+this APE will be invited to be in the first group of Voting Members:
+
+* has a named role in the project and has participated in an Astropy
+  coordination meeting in the last two years,
+* has a named role in the project and has regularly participated in Astropy
+  telecons (e.g., co-working telecon, spectroscopy sprint, Astropy Learn
+  telecon) in the last two years,
+* has commit rights to at least one repository in the astropy GitHub
+  organization and has actively used those commit rights in the last two years.
```

Comment:
I agree that we shouldn't include some of the few random repositories that ended up in the organisation but aren't actually maintained by the project (and we should probably tidy this up at some point)

Reply:
I agree with this in principle except that I thought we (although by "we" I thought it was either @astrofrog, @bsipocz, or @Cadair !) checked this during the WG meeting and it turns out that applying these criteria don't yield anyone particularly unexpected - i.e. making small changes like this wouldn't actually change the list of names.

Reply:
it was certainly not me, I would have pointed this out there and then had I had the chance. 

And even if it doesn't change much, it is very unfair to count write access a subset of the affiliated but not coordinated packages while ignore the other 40 ones. It is possible that people in the WG didn't know that there are non astropy project packages under the astropy github organization when voted for this phrasing. I think everyone should be aware of that structure as one of the fundamentals of how astropy works, but of course that wasn't a criteria to be part of the WG.

Reply:
If one aims for work for astropy, I guess one should also include the `liberfa` repositories. That might add one or two for the initial list.

Reply:
What about wcslib and cfitsio and numpy then? 

If astropy officially maintains liberfa (similarly to the coordinated affiliated packages) and thus it should be treated differently than the other dependencies, then it should be moved under the astropy github org and treated as part of the project.

Reply:
(if it's not clear, I would be 👍 to make that move, just would like to see the consistency by having it under the same parent org)

Reply:
`liberfa` is definitely primarily a spin-off of astropy - and, yes, I think people working on other packages to make sure that they interoperate with astropy should count too.

Anyway, rereading I see this was really just about the initial list, and the people working at liberfa are mostly already on that for other reasons, so perhaps somewhat irrelevant.

Reply:
I assumed liberfa was under the astropy organization already.  Shouldn't it be?

Reply:
I feel if liberfa maintainership comes with astropy voting rights, then it should indeed live in the astropy organization. I know that future members will be voted in independently of this current list, but even then if someone is very active in an upstream library, it shouldn't really be enough to get the voting right, activity within the astropy project is what should count.

Reply:
(and thus I think liberfa belongs to the astropy org than the current, random selection of affiliated packages that reside in astropy for various reasons)

Reply:
@bsipocz - indeed, in terms of relevance liberfa belongs in astropy more than some of the random things on github. That said, I see little use in moving it to the astropy umbrella.

But I regret somewhat bringing it up. Overall, I think it actually is better to not try to nail down what constitutes real contribution in this APE, or at least not for beyond the initial list. In that sense, "contribute meaningfully" should perhaps be good enough, as it is voted on anyway.

Reply:
I'm very happy that you brought this up though, as I think I would prefer less flowery language all over and more concrete examples. E.g "contribute meaningfully" is very subjective and can lead easily that friends get voted in just because they are nominated and no one cares or dares to speak up, while people outside of the usual network will have a much harder time to get recognition for a sustained, good level of contributions (happened in so many cases with the current roles setup).

Reply:
@bsipocz - point about concrete examples taken - and very much agreed with. So maybe the solution here is then to make clear that the list of concrete examples is not meant to be exhaustive (for contributors beyond the initial list, i.e., those who get voted on).

Reply:
Definitely open to more concrete examples if that would be helpful, and agreed something about it not being exhaustive makes sense.  Please suggest some concrete examples and I can try to integrate them if others agree they are valid!

That said, the point the WG consensus was getting at here is that the underlying problem is that there is *no*  objective way the WG could agree on to measure this, because it's too hard to map any objective metric (e.g. "commits" or even "hours") onto anything anyone was willing to say was any more objective than this scheme.

So the idea here is that the voting members are not voting their "friends" or "people they want to hang out with", but rather the voting process is saying "people who meet those criteria at the top of the membership section should be voting members, and we are voting that, to the best of our knowledge, this person has met those criteria".  Is there some way that nuance could be made more clear?

Reply:
See my comment on the thread above as to my thoughts on how to judge this list of criteria. If there are active repos under the astropy org which lead to people being selected by this process which don't meet the criteria in the membership section above then it needs reworking, otherwise it's serving it's purpose.

Reply:
See https://github.com/astropy/astropy-APEs/pull/61#issuecomment-769284716 for new wording for the rules for voting members - I now explicitly say it has to be the core repo, coordinated repos, infrastructure, or learn repos.